src/Components/Blazor/Blazor/src/Microsoft.AspNetCore.Blazor.csproj

```diff
@@ -10,6 +10,9 @@
     <Reference Include="Mono.WebAssembly.Interop" />
     <Reference Include="Microsoft.AspNetCore.Components.Web" />
     <Reference Include="Microsoft.Extensions.Options" />
+
+    <!-- This project does not yet include the latest changes from the 'blazor-wasm' branch. -->
```

Comment:
TBH it's not clear to me from this comment what problem occurs or why the line below fixes it. Any chance it could be rephrased, perhaps to link to whatever issue it's addressing?

Approving anyway so you can merge at your convenience :)

Reply:
FYI warning is `BUILD001: Reference to 'Microsoft.Extensions.Configuration' was removed since the last stable release of this package.` without the suppression. I'll change the comment to just mention that.